Libplanet.Net/BoundPeer.cs

```diff
@@ -81,15 +81,20 @@ public static BoundPeer ParsePeer(string peerInfo)
             }
 
             string[] tokens = peerInfo.Split(',');
-            if (tokens.Length != 3)
+            if (tokens.Length != 4)
             {
                 throw new ArgumentException(
-                    $"'{peerInfo}', should have format <pubkey>,<host>,<port>",
+                    $"'{peerInfo}', should have format <pubkey>,<pubkeytype>,<host>,<port>",
```

Comment:
As this would break the API anyway, I think type prefix might be better, i.e. `<pubkeyType>,<pubkey>,<host>,<port>`.

Reply:
Amended! https://github.com/planetarium/libplanet/pull/2281/commits/3e40c02fca04d742595b0a3cb6e6083e1b806244